.travis.yml

```diff
@@ -2,14 +2,19 @@ language: java
 
 jdk:
   - oraclejdk8
-#  - openjdk11
+  - openjdk8
+  - openjdk11
 
-addons:
-  apt:
-    packages:
-      - oracle-java8-installer # Updates JDK 8 to the latest available.
+before_install:
+  - mvn -N io.takari:maven:wrapper -Dmaven=3.6.0
```

Comment:
Why use `mvnw`?

Reply:
For Java 11, using the most recent Maven and Maven Plugin versions is the most likely to be compatible.  This command is running with existing maven provided by travis.  It downloads Maven 3.6.0 so we have exact control over what to run with and installs it like the gradlew wrapper in the project.  Hence, mvnw after this point.